lib/pages/pool/MyLiqudity/GetTokenPairIcons.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:ax_dapp/service/Controller/Token.dart';
+import 'package:ax_dapp/service/TokenList.dart';
+
+class GetTokenPairIcons {
```

Comment:
Didn't we have a class that returns two token objects based on addresses passed? Tickers are kind of iffy...